feature-libs/product/configurators/rulebased/cpq/cpq.models.ts

```diff
@@ -123,4 +124,12 @@ export namespace Cpq {
     INPUT = 95,
     AUTO_COMPLETE_CUSTOM = 102,
   }
+
+  export enum DataType {
+    INPUT_STRING = 'String',
+    INPUT_NUMBER = 'Number',
+    QTY_ATTRIBUTE_LEVEL = 'Quantity',
+    QTY_VALUE_LEVEL = 'Attr.Quantity',
+    N_A = 'N/A',
```

Comment:
this seems to be a duplicate of Configurator.DataType?

Reply:
Enum values in the Configuration.models.ts are renamed